extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/runtime/LiquibaseMongodbConfig.java

```diff
@@ -33,6 +40,13 @@ public class LiquibaseMongodbConfig {
     @ConfigItem
     public boolean cleanAtStart;
 
+    /**
+     * true to quit exit right after the operation.
+     *
+     */
+    @ConfigItem
+    public boolean runAndExit;
```

Comment:
Same question as the Flyway extension

Reply:
Acts as a flag to turn the job off, when we externalize it.

Reply:
Added comment